deploy/scripts/src/ipv-core/test.ts

```diff
@@ -119,6 +121,7 @@ export function coreScenario1 (): void {
           tags: { name: 'B01_Core_01_LaunchOrchestratorStub' }
         }
       )
+      if (env.staticResources) getStaticResources(res)
```

Comment:
Is it possible to include some kind of `check` to validate that the `http.batch` request was successful?

Reply:
As discussed, adding a check specifically for the http.batch request would be too noisy. If any of the request fail, it would be added to the http_req_failed metric.
Closing the comment.